packages/http/libraries/core/src/http/adapter/InversifyHttpAdapter.ts

```diff
@@ -29,9 +30,14 @@ export abstract class InversifyHttpAdapter<
   TNextFunction extends (err?: unknown) => void,
 > {
   readonly #container: Container;
+  readonly #httpAdapterOptions: HttpAdapterOptions;
```

Comment:
```suggestion
  readonly #httpAdapterOptions: Required<HttpAdapterOptions>;
```

I would love to avoid dealing with nullish internal properties, does it make sense? For now `Required` is good enough, but we can have a better type once we need it.

Reply:
I understand, but there will be situations where we want optional properties to have default behaviors, as is the case here. It seems to me a more user-friendly approach.

Reply:
Indeed, let's keep the interface for the user, but let's internally use another with non nullish values :)